### DISCUSSION TITLE
Numerical Instability

### DISCUSSION THREAD

**Discussion by hunzideriji** [firemodels/fds#13068] [Q&A] [answered]
When the grid is large, there is no numerical instability, why is the numerical instability when the mesh is small?
Here's my code
Please help me with an answer, thanks.

&MESH ID='Mesh01', IJK=128,40,120, XB=-0.02,0.62,0.4,0.6,-0.1,0.5/

[新建文本文档 (2).txt](https://github.com/user-attachments/files/15863724/2.txt)

**Comment by drjfloyd** [comment]
You did not attach a complete input file. There is no MESH or TIME line in the file. Please attach a complete input file and also clearly indicate the successful and unsuccessful MESH inputs.

**Reply by hunzideriji** [reply]
Sorry. Here's my code, I built the model on pyrosim, and the code about time can't be copied, so I took a screenshot, 
When I set the grid to a square with an edge length of 5mm, the values are unstable. When my grid is set to a square with an edge length of 10mm, this problem does not occur 
thank you
![1718632847950](https://github.com/firemodels/fds/assets/173037310/6f2cb4e6-92f2-403a-a66f-76a457c6378e)

[新建文本文档.txt](https://github.com/user-attachments/files/15872122/default.txt)

**Reply by drjfloyd** [reply]
When you run a case in Pyrosim it writes an FDS file and then starts a shell an runs FDS with that input. The input file should be on your drive in the folder with your pyrosim file.

**Reply by hunzideriji** [reply]
Yes, I found it.
[新建文本文档.txt](https://github.com/user-attachments/files/15878899/default.txt)


This is my code. When the grid is a cube with a side length of 0.005 meters, there will be error problems caused by numerical instability. When the grid is a cube with a side length of 0.02m, it can operate normally. May I ask why such a problem occurs.
thank you.

**Comment by drjfloyd** [comment, marked as answer]
The case dies with the current release and looking at the plot3d file that is dumped, there is a region of very high heat release rate per unit volume.   Your input file has this surface:

&SURF ID='Surface01',
      RGB=26,204,26,
      VOLUME_FLOW=-1.0E-5,
      TAU_V=-1.0,
      MASS_FRACTION=1.0,
      SPEC_ID='PROPANE',
      TAU_MF=1.0/

This emits a fuel air mixture for the first second. I changed to TAU_MF=0 so it is always pure fuel like HRRPUA does. The case got past the point of failure. This is suggesting that at the fine grid resolutions the premixed condition was resulting in too high of a volumetric heat release rate.  @rmcdermo you may want to look at this case.

**Reply by hunzideriji** [reply]
Thank you very much for your help
